tap_chargebee/streams/base.py

```diff
@@ -195,30 +195,38 @@ def sync_data(self):
                     break
 
             records = response.get('list')
-            
-            to_write = self.get_stream_data(records)
-            
+
+            # list of deleted records
```

Comment:
Add a detailed comment like "List of deleted plans, addons, and coupons from the events endpoint"

Reply:
Added comment as suggested.